src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+ceasing-to-be&mdash; which currently is defined with the vanishing of `being`
+into `nothing` and `nothing` into `being` respectively&mdash;could be equally
+applied over the category `nothing` itself. That is to say, `nothing` could be
+coming-to-be where _its_ self-subsistence _as nothing_ concerned and
+creasing-to-be when _it_ vanishes into `being`. This idea, while tempting at
+first, is not really coherent. Firstly, it presupposes the original coming-to-be
+and ceasing-to-be vis-à-vis `being` and `nothing`; secondly, it breaks the
+definition of `nothing` as the absolute absence of `being`, which is required
+for the original movement. Therefore, the coming-to-be and ceasing-to-be cannot
+apply to `nothing` except in a very abstract or non-substantive sense.
+
+#### Mutual Paralysis
+
+> Both are the same, becoming, and even as directions that are so different they
+> interpenetrate and paralyze each other. The one is _ceasing-to-be_; being
+> passes over into nothing, but nothing is just as much the opposite of itself,
+> the passing-over into being, coming-to-be. This coming-to-be is the other
+> direction; nothing goes over into being, but being equally sublates itself and
+> is rather the passing-over into nothing; it is ceasing-to-be. &ndash; They do
+> not sublate themselves reciprocally &ndash; the one sublating the other
+> externally &ndash; but each rather sublates itself in itself and is within it
+> the opposite of itself (Hegel 2010, 80-1/21.93).
+
+While the two specific movements of coming-to-be and ceasing-to-be point in
+opposite directions, they fold into same `becoming` and come to mirror the
+initial inseparability that defined `being` and `nothing`.
+
+Importantly, the source of each specific movement _is_ in the root category in
+question and there is no external action or cause that the made the category
+operate in the way that it did, namely, that it vanished into its other. Putting
+it plainly, `being` _is_ ceasing-to-be as it vanishes into `nothing`, but
+likewise `nothing` _is_ coming-to-be as it vanishes into `being`.
+
+In addition, Hegel writes that the two specific movements interpenetrate and
+paralyze each other. How do they relate and why would they paralyze each other?
+
+In the first place, the two specific movements relate because they are of the
+same inseparability; the same `becoming` defines coming-to-be and ceasing-to-be.
+Once this is understood, the issue with their opposite movements should become
+clear, namely, the same `becoming` is a movement _both_ of coming-to-be _and_
+ceasing-to-be. Recall that for `pure being` and `nothing`, their vanishing is
+total and absolute, and occurs immediately. Indeed, the vanishing is so
+immediate that Hegel indicates it by stating it is
+[not passing but has passed](#the-unity-of-being-and-nothing). As the categories
+are this unstable, effectively coming-to-be _is_ ceasing-to-be, and
+ceasing-to-be _is_ coming-to-be&mdash;emergence and cessation occur
```

Comment:
And I think this supports the above reading, that nothing can cease-to-be and being can come-to-be

Reply:
Okay, let's keep the thread going in the previous comment 🙂